src/data/fintype/basic.lean

```diff
@@ -871,6 +871,8 @@ fintype.of_equiv _ equiv.plift.symm
   fintype.card (plift α) = fintype.card α :=
 fintype.of_equiv_card _
 
+instance (α : Type*) [fintype α] : fintype (order_dual α) := ‹fintype α›
```

Comment:
Does `@[derive fintype]` on `order_dual` work instead of manually doing this?

Reply:
How so? `order_dual` is defined way earlier than `fintype`.

Reply:
Update: `attribute [derive fintype] order_dual` fails.

Reply:
Okay, thanks for trying!